openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -302,7 +302,6 @@ class AccountCreationActivationAndPasswordChangeTest(TestCase):
         '@domain.com',
         'test@no_extension',
         u'fŕáńḱ@example.com',
-        u'frank@éxáḿṕĺé.ćőḿ',
```

Comment:
Interesting, I didn't know. "In May 2010 the first IDN ccTLD were installed in the DNS root zone"
https://en.wikipedia.org/wiki/Internationalized_domain_name